draft-ietf-quic-tls.md

```diff
@@ -1116,9 +1125,22 @@ TLS KeyUpdate message.  Endpoints MUST treat the receipt of a TLS KeyUpdate
 message as a connection error of type 0x10a, equivalent to a fatal TLS alert of
 unexpected_message (see {{tls-errors}}).
 
-An endpoint MUST NOT initiate more than one key update at a time.  A new key
-cannot be used until the endpoint has received and successfully decrypted a
-packet with a matching KEY_PHASE.
+An endpoint MUST NOT initiate the first key update until the handshake is
+confirmed from its perspective. An endpoint MUST NOT initiate a subsequent
+key update until it has received an ACK for a packet sent at the previous
+KEY_PHASE.  This can be implemented as checking whether
+largest_recvd_ack_1rtt >= last_key_update_pn, where largest_recvd_ack_1rtt
```

Comment:
I think that I prefer words.  "This requires tracking the lowest packet number sent with updated keys.  Once an acknowledgment for that packet number or any higher packet number is received, another key update can be initiated."

Reply:
Switched to using words